templates/blog/macros.html

```diff
@@ -1,6 +1,8 @@
 {% macro article_list(articles, with_author=true) %}
+  <ul>
   {% for article in articles %}
   {% set url = url_for_article(article) %}
+  <li>
```

Comment:
I'm still not confident we need `ul > li > article` and not just a serie of `article`, but if you have good reasons I'll rely on you.